community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/scripts/slurmsync.py

```diff
@@ -99,9 +97,8 @@ def _find_dynamic_node_status() -> NodeStatus:
     return NodeStatus.unchanged  # don't touch dynamic nodes
 
 
-def _find_tpu_node_status(nodename, state):
```

Comment:
Why change this interface?

Reply:
The callee now has access to each nodeset, before this function would find the nodeset but now there is no need since it can just be passed into this function directly